tests/manage/pv_services/test_del_mon_service_and_create_pvc.py

```diff
@@ -235,3 +237,91 @@ def test_pvc_creation_after_del_mon_services(self, interface, pod_factory):
         log.info(f"Create {interface} PVC")
         pod_obj = pod_factory(interface=interface)
         pod_obj.run_io(storage_type="fs", size="500M")
+
+    @bugzilla("1969733")
+    @pytest.mark.polarion_id("OCS-2611")
+    def test_del_mon_svc(self, multi_pvc_factory):
```

Comment:
this test looks quite disruptive so in case it fails in the middle, there is no teardown to bring the cluster back to health state. Please add such a teardown

Reply:
added teardown function to validate mon services are up or not, if not restart the operator pod.